config/printer-creality-cr10s-pro-v2-2020.cfg

```diff
@@ -0,0 +1,158 @@
+# This file contains pin mappings for the Creality CR-10S Pro V2. To use
+# this config, the firmware should be compiled for the AVR atmega2560.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+## General Config
+[mcu]
+serial: /dev/serial/by-id/<YOUR_USB_ID>
+
+[printer]
+kinematics: cartesian
+max_velocity: 200
+max_accel: 1500
+max_z_velocity: 10
+max_z_accel: 100
+
+[gcode_macro G29]
```

Comment:
Please do not include macros in a config that do not pertain to base functionality of a printer, BED_MESH_CALIBRATE is perfectly valid gcode in its own right.

Thanks
James

Reply:
Done